core/templates/dev/head/pages/about/about.html

```diff
@@ -26,13 +26,13 @@ <h1 translate="I18N_ABOUT_PAGE_HEADING"></h1>
         <div class="about-nav md-tab">
           <ul class="oppia-about-tabs">
             <li class="oppia-about-tabs-active">
-              <a class="oppia-about-tabs-text" href="about" translate="I18N_ABOUT_PAGE_TABS_ABOUT"></a>
+              <a ng-click = "onAboutOppiaClick('about')" href="" class="oppia-about-tabs-text" id="about" translate="I18N_ABOUT_PAGE_TABS_ABOUT"></a>
```

Comment:
Haha will do, sorry. I'm a contributor to MediaWiki software, and this is probably just me doing it without thinking. Thanks!